gordo_components/model/models.py

```diff
@@ -761,7 +626,7 @@ def fit(
     ) -> "KerasLSTMForecast":
 
         """
-        This fits a one step forecast LSTM architecture using Keras fit_generator method.  For further details
+        This fits a one step forecast LSTM architecture.  For further details
         on the input/output samples fed to this method, refer to the generate_window method.
```

Comment:
Update docstring (generate_window -> get_TimeseriesGenerator) - same for KerasLSTMAutoencoder.